pkg/batch/jobs/calculator/experiment_calculate.go

```diff
@@ -52,6 +52,7 @@ type experimentCalculate struct {
 
 func NewExperimentCalculate(
 	httpStan *stan.Stan,
+	modelID string,
```

Comment:
Please rename `modelID` -> `stanModelID` here and in the other places.
```suggestion
	stanModelID string,
```